src/data.ts

```diff
@@ -552,4 +553,105 @@ export function fetchTopLevelQueries(selectedCountryCode): Promise<Map<string, Q
       }
     );
   return topQueriesData;
+}
+
+export interface ClusterRow {
+  date: string;
+  country_region: string;
+  country_region_code: string;
+  sub_region_1: string;
+  sub_region_1_code: string;
+  sub_region_2: string;
+  sub_region_2_code: string;
+  sub_region_3: string;
+  sub_region_3_code: string;
+  place_id: string;
+  query_type: string;
+  query: string;
+  rank: number;
+  sni: number;
+  history: string;
+  members: string;
+  num_members: number;
+  category: string;
+}
+
+export interface Cluster {
+  query: string;
+  sni: number;
+  rank: number;
+  change: number;
+  members: string[];
+}
+
+function removeDuplicate(clusterQuery: string, members: string[]): string[] {
+  return members.filter(member => member != clusterQuery)
+}
+
+function calculateChange(history: number[], current: number) {
+  if (history.length === 0) {
+    return null
+  }
+  const previous = history[history.length - 1];
+  return Math.round(((current - previous) / previous) * PERCENTAGE);
+}
+
+function createCluster(clusterRow: ClusterRow): Cluster {
+  const historyList = clusterRow.history === "" ? [] : clusterRow.history.split("|").map(value => Number(value));
+  const membersList = clusterRow.members === "" ? [] : removeDuplicate(clusterRow.query, clusterRow.members.split("|"));
+  const clusterChange = calculateChange(historyList, clusterRow.sni);
+  return { query: clusterRow.query, sni: clusterRow.sni, rank: clusterRow.rank, change: clusterChange, members: membersList };
+}
+
+let clustersStoragePrefix: string = "https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/staging/top_clusters/";
```

Comment:
remove "staging" from the url

Reply:
Removed and can confirm that everything is showing up as expected!

![image](https://user-images.githubusercontent.com/26349829/167055278-34cbaa88-6540-41fe-aad6-a714f6101956.png)